documents/charter.md

```diff
@@ -7,11 +7,13 @@ influence these applications. However, when networks enforce
 rate-limiters, applications like video streaming or conferencing
 struggle to adapt, leading to a suboptimal user experience.
 
-This WG aims to establish a mechanism for devices intending to
+This WG aims to establish a mechanism for network devices intending to
```

Comment:
```suggestion
This WG aims to establish a mechanism for network devices intending to
```
I think "capable of" might be preferable - you say "capable of" in the next change - but overall, this looks fine to me. 

Thank you for leaping onto the issue and writing a PR!